docs/src/main/asciidoc/kubernetes-client.adoc

```diff
@@ -53,9 +53,9 @@ quarkus.kubernetes-client.trust-certs=false
 quarkus.kubernetes-client.namespace=default
 ----
 
-Note that the full list of properties is available in the https://github.com/quarkusio/quarkus/blob/main/extensions/kubernetes-client/runtime-internal/src/main/java/io/quarkus/kubernetes/client/runtime/KubernetesClientBuildConfig.java[KubernetesClientBuildConfig] class.
+Note that the full list of properties is available in the link:#quarkus-kubernetes-client_quarkus.kubernetes-client.devservices-dev-services[Dev Services section of the configuration reference].
 
-In dev mode and when running tests, xref:kubernetes-dev-services.adoc[Dev Services for Kubernetes] automatically starts a Kubernetes API server.
+In dev mode and when running tests, xref:kubernetes-dev-services.adoc[Dev Services for Kubernetes] automatically start a Kubernetes API server.
```

Comment:
I am not totally sure about this one. I understand why you made the change, but the way I read it is that `Dev Services for Kubernetes` is a single entity, therefore `starts` is appropriate

Reply:
Yeah I must admit I hesitated a bit about that one. So if you also do, let's keep it as is. I'll adjust the PR.